common/resource/fx.go

```diff
@@ -56,6 +56,7 @@ import (
 	"go.temporal.io/server/common/membership"
 	"go.temporal.io/server/common/metrics"
 	"go.temporal.io/server/common/namespace"
+	"go.temporal.io/server/common/namespace/registry"
```

Comment:
Can we just give this a clearer name locally? (not rename the package itself)
```suggestion
	nsregistry "go.temporal.io/server/common/namespace/registry"
```

I'd say we should do this wherever it's imported (like our *pb convention). It's not that many places, right?

Reply:
Would it be best to change the package name as a whole then?

Reply:
We chatted elsewhere, we chose to rename to nsregistry